TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/datahandlers/http/helpers/IHttpMessageAdapter.java

```diff
@@ -0,0 +1,50 @@
+package org.opensearch.migrations.replay.datahandlers.http.helpers;
```

Comment:
This class would be easier to immediately comprehend and understand its purpose if it were an inner class of SigV4AuthTransformer

Reply:
Moved